src/workerd/api/actor.h

```diff
@@ -20,14 +20,45 @@ namespace workerd {
 
 namespace workerd::api {
 
+enum IsInHouse {
+  YES,
+  NO,
+};
+
+// TODO(soon): This will eventually become the Proxy object.
+// This class extends Fetcher with the JS RPC call function, but doesn't go as far as DurableObject.
+// We don't want to define RPC stuff on Fetcher itself, and since ColoLocalActorNamespace::get()
```

Comment:
> We don't want to define RPC stuff on Fetcher itself

Why not? I'd like RPC to work for service bindings, too.

Reply:
Yeah, me too but we can always move it up to its own concept when that time comes.